vendor/archive/tar/LICENSE

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2009 The Go Authors. All rights reserved.
```

Comment:
Ah, hm, looks like something still went wrong; that's an issue with the latest version of vndr, which removes everything before vendoring.

The `archive/tar` vendor should not be removed